fury/actor.py

```diff
@@ -430,6 +439,11 @@ def contour_from_roi(data, affine=None,
     skin_actor.GetProperty().SetColor(color[0], color[1], color[2])
     skin_actor.GetProperty().SetOpacity(opacity)
 
+    if pbr:
+        skin_actor.GetProperty().SetInterpolationToPBR()
+        skin_actor.GetProperty().SetMetallic(metallic)
+        skin_actor.GetProperty().SetRoughness(roughness)
```

Comment:
It will be good if this block should be a new function with `actor, metallic and roughness` as a parameter

Reply:
I've made the suggested changes and added the tests on the currently implemented actors.